### PR TITLE
fix token in summary

### PR DESCRIPTION
**Pull request by yishengjin1413**
# Fix token in summary

To fix CSI bug 966058:[.NET API Browser] Token is not shown correctly in description of API browser page, flattened the token reference in summary node under docs.

### CODE REVIEW

File: xml/Microsoft.Windows.Themes/ButtonChrome.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Creates the theme-specific look for [!INCLUDE[TLA#tla_avalonwinfx](~/includes/tlasharptla-avalonwinfx-md.md)] <see cref="T:System.Windows.Controls.Button" /> elements.</summary>
+    <summary>Creates the theme-specific look for Microsoft .NET Framework <see cref="T:System.Windows.Controls.Button" /> elements.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Windows.Themes/ButtonChrome.xml]: Microsoft .NET Framework --> .NET Framework

File: xml/Microsoft.Windows.Themes/ListBoxChrome.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Creates the theme-specific look for [!INCLUDE[TLA#tla_avalonwinfx](~/includes/tlasharptla-avalonwinfx-md.md)] <see cref="T:System.Windows.Controls.ListBox" /> elements.</summary>
+    <summary>Creates the theme-specific look for Microsoft .NET Framework <see cref="T:System.Windows.Controls.ListBox" /> elements.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Windows.Themes/ListBoxChrome.xml]: Microsoft .NET Framework --> .NET Framework

File: xml/Microsoft.Windows.Themes/PlatformCulture.xml [hunk 3]
```diff
@@ -31,7 +31,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Provides culture-specific information used by the [!INCLUDE[TLA#tla_avalonwinfx](~/includes/tlasharptla-avalonwinfx-md.md)] system themes.</summary>
+    <summary>Provides culture-specific information used by the Microsoft .NET Framework system themes.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Windows.Themes/PlatformCulture.xml]: Microsoft .NET Framework --> .NET Framework

File: xml/Microsoft.Windows.Themes/ScrollChrome.xml [hunk 4]
```diff
@@ -27,7 +27,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Creates the theme-specific look for [!INCLUDE[TLA#tla_avalonwinfx](~/includes/tlasharptla-avalonwinfx-md.md)] <see cref="T:System.Windows.Controls.Primitives.ScrollBar" /> elements.</summary>
+    <summary>Creates the theme-specific look for Microsoft .NET Framework <see cref="T:System.Windows.Controls.Primitives.ScrollBar" /> elements.</summary>
```
Comment (rpetrusha) [inline xml/Microsoft.Windows.Themes/ScrollChrome.xml]: Microsoft .NET Framework --> .NET Framework

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for flattening these tokens, @yishengjin1413. Would you mind making a four changes? "Microsoft .NET Framework" should simply be ".NET Framework".

**Comment by mairaw** [comment]
I've edited the files via GitHub UI so we can merge this after builds are completed successfully.

**Review by mairaw — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**